client_secret.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The secret file should not be publicly accessible, and should be part of the `.gitignore`.

Reply:
done